python/cudf/cudf/pandas/_wrappers/pandas.py

```diff
@@ -429,6 +429,21 @@ def Index__setattr__(self, name, value):
     },
 )
 
+# Special caseing `ArrowDtype` as it is not yet added to `cudf` namespace
+# both fast and slow paths are `pd.ArrowDtype`
+ArrowDtype = make_final_proxy_type(
+    "ArrowDtype",
+    pd.ArrowDtype,
+    pd.ArrowDtype,
```

Comment:
Shouldn't these be `_Unusable`?

Reply:
Ideally they should be, but since we support `pd.ArrowDtype` all across our code-base in `cudf` when pandas-compatibility mode is enabled, this is just a temporary workaround until we add `cudf.ArrowDtype` namespace.

Reply:
Is there an open issue to track?